client/components/product-list-item.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+export default class ProductionListItem extends React.Component {
+  render() {
+
+    return (
+      this.props.products.map((product, index) =>
+
+        <div className="col-4 mb-4" key={index}>
+          <div className="card">
+            <img src={product.image} alt={product.name} className="card-img-top" />
+            <div className="card-body">
+              <h5 className="card-title">{product.name}</h5>
+              <p className="card-text">${(product.price / Math.pow(10, 2)).toFixed(2)}</p>
+              <p className="card-text">{product.shortDescription}</p>
+            </div>
+          </div>
+
+        </div>
+
+      ));
```

Comment:
Ok, so the mapping should happen in the list component. Take a look at `fetch-in-react`, `fetch-in-react-practice`, and `sgt-react`. The single-item components are only responsible for showing one thing and the list components are responsible for mapping through the data array.